www/webapp/main.py

```diff
@@ -23,6 +23,10 @@ def register():
     return render_template("register.html")
 
 
-@webapp.route('/login')
-def login():
-    return render_template("login.html")
+@webapp.route('/signup')
+def signup():
+    return render_template("signup.html")
+
+@webapp.route('/pricing')
+def pricing():
+    return render_template("pricing.html")
```

Comment:
This is not the place for this. Please add sign in/sign up page to the auth module.

For now remove pricing route.